symbolic-symcache/tests/test_writer.rs

```diff
@@ -351,3 +351,25 @@ fn test_trailing_marker() -> Result<(), Error> {
 
     Ok(())
 }
+
+/// Tests that addresses between functions are unmapped. See
+/// <https://github.com/getsentry/symbolic/pull/897>.
+#[test]
+fn test_lookup_between_functions() {
+    let buffer = ByteView::open(fixture(
+        "macos/3CD3E3CC-281E-3AF3-84EB-CDE6D56F0559.dSYM/Contents/Resources/DWARF/CrashLibiOS",
```

Comment:
can you only commit the actual dwarf payload and remove all the other `dSYM` junk thats not useful?

also, that file is a ~1.5M. is it possible to slim that down somehow?

Reply:
I decided to just reuse the Python fixture instead of committing a copy. Didn't pay attention to the file size before.